shared/constants/types/fs.js

```diff
@@ -43,9 +43,10 @@ export type State = I.RecordOf<_State>
 
 export type Visibility = 'private' | 'public' | 'team' | null
 
-export const stringToPath = (s: string): Path => (s.indexOf('/keybase') !== -1 ? s : null)
+export const stringToPath = (s: string): Path => (s.indexOf('/') === 0 ? s : null)
```

Comment:
Is there a place we are handling paths not starting with `/keybase` now?

Reply:
Oh I see the `pathConcat` below needs this now. I thought making sure it starts with `/keybase` was a nice guarantee. Is there another way we could have it?

Reply:
meh I don't think it's necessary. You're right it was kind of nice but in the end I think it may have been too ambitious.